client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+const options = require("./options");
+const utils = require("./utils");
+const sorting = require("./sorting");
+
+const chat = $("#chat");
+const sidebar = $("#sidebar, #footer");
+
+module.exports = {
+	buildChannelMessages,
+	buildChatMessage,
+	renderChannel,
+	renderChannelMessages,
+	renderChannelUsers,
+	renderNetworks
+};
```

Comment:
Another no-big-deal question: I was under the impression that it was fairly common/standard to add the `exports` statement at the end of the JS files, am I incorrect?